### DISCUSSION TITLE
Does Xenia Manager install Xenia in portable mode, or is it possible to do so?

### DISCUSSION THREAD

**Discussion by Bobble80** [xenia-manager/xenia-manager#167] [Q&A] [answered]
Hi, I was looking to try out Xenia Manager for the option to have per game settings and profiles, seems like a great option that Xenia standalone is lacking. 
But I was wondering if I was to install Xenia Canary in Manager, does it install it in portable mode or is there a way for me to install it in portable mode?

I always like to have my emulators set up as portable installs and currently use Canary in portable mode now. I’m just not sure how Manager works and if this is possible or not.

Can anyone let me know about this? Thanks

**Comment by shazzaam7** [comment, marked as answer]
Everything is portable except the locations to the game .xex/.iso etc.
